map_name: map_package/Maps/bigmap/bigmap

regions:
- id: region_1
  speed_gain: 1.0
  steering_gain: 1.0
  corners:
    top_left:
    - 58.67666666666668
    - -0.22193127687473746
    top_right:
    - 44.46666666666667
    - -0.22193127687473746
    bottom_right:
    - 44.46666666666667
    - -13.37627902209962
    bottom_left:
    - 58.67666666666668
    - -13.37627902209962
  center:
  - 51.57166666666667
  - -6.799105149487179
  width: 14.210000000000008
  height: 13.154347745224882
- id: region_2
  speed_gain: 1.0
  steering_gain: 1.0
  corners:
    top_left:
    - 11.11666666666666
    - 2.679763078689575
    top_right:
    - -11.309999999999995
    - 2.679763078689575
    bottom_right:
    - -11.309999999999995
    - -12.699217005801282
    bottom_left:
    - 11.11666666666666
    - -12.699217005801282
  center:
  - -0.09666666666666757
  - -5.0097269635558535
  width: 22.426666666666655
  height: 15.378980084490857
- id: region_3
  speed_gain: 1.0
  steering_gain: 1.0
  corners:
    top_left:
    - 60.126666666666665
    - -43.26373088441204
    top_right:
    - 47.656666666666666
    - -43.26373088441204
    bottom_right:
    - 47.656666666666666
    - -54.773785161483815
    bottom_left:
    - 60.126666666666665
    - -54.773785161483815
  center:
  - 53.891666666666666
  - -49.01875802294793
  width: 12.469999999999999
  height: 11.510054277071774
- id: region_4
  speed_gain: 1.0
  steering_gain: 1.0
  corners:
    top_left:
    - 36.92666666666668
    - -42.97356144885561
    top_right:
    - 22.813333333333333
    - -42.97356144885561
    bottom_right:
    - 22.813333333333333
    - -54.96723145185477
    bottom_left:
    - 36.92666666666668
    - -54.96723145185477
  center:
  - 29.870000000000005
  - -48.970396450355196
  width: 14.113333333333344
  height: 11.99367000299916
- id: region_5
  speed_gain: 1.0
  steering_gain: 1.0
  corners:
    top_left:
    - 32.769999999999996
    - -24.015824992502104
    top_right:
    - 19.816666666666663
    - -24.015824992502104
    bottom_right:
    - 19.816666666666663
    - -35.52587926957388
    bottom_left:
    - 32.769999999999996
    - -35.52587926957388
  center:
  - 26.29333333333333
  - -29.77085213103799
  width: 12.953333333333333
  height: 11.510054277071774
- id: region_6
  speed_gain: 1.0
  steering_gain: 1.0
  corners:
    top_left:
    - 10.053333333333327
    - -20.340345475453972
    top_right:
    - -9.763333333333335
    - -20.340345475453972
    bottom_right:
    - -9.763333333333335
    - -32.81763120438052
    bottom_left:
    - 10.053333333333327
    - -32.81763120438052
  center:
  - 0.14499999999999602
  - -26.578988339917245
  width: 19.816666666666663
  height: 12.477285728926546
- id: region_7
  speed_gain: 1.0
  steering_gain: 1.0
  corners:
    top_left:
    - -21.17
    - -22.7584241050909
    top_right:
    - -32.96333333333334
    - -22.7584241050909
    bottom_right:
    - -32.96333333333334
    - -36.106218140686735
    bottom_left:
    - -21.17
    - -36.106218140686735
  center:
  - -27.06666666666667
  - -29.432321122888816
  width: 11.793333333333337
  height: 13.347794035595836
- id: region_8
  speed_gain: 1.0
  steering_gain: 1.0
  corners:
    top_left:
    - -23.78
    - -45.294916933307064
    top_right:
    - -35.86333333333333
    - -45.294916933307064
    bottom_right:
    - -35.86333333333333
    - -54.87050830666929
    bottom_left:
    - -23.78
    - -54.87050830666929
  center:
  - -29.821666666666665
  - -50.08271261998817
  width: 12.083333333333329
  height: 9.575591373362222
- id: region_9
  speed_gain: 1.0
  steering_gain: 1.0
  corners:
    top_left:
    - -47.85
    - -44.52113177182324
    top_right:
    - -58.193333333333335
    - -44.52113177182324
    bottom_right:
    - -58.193333333333335
    - -54.87050830666929
    bottom_left:
    - -47.85
    - -54.87050830666929
  center:
  - -53.02166666666667
  - -49.695820039246264
  width: 10.343333333333334
  height: 10.349376534846044
- id: region_10
  speed_gain: 1.0
  steering_gain: 1.0
  corners:
    top_left:
    - -45.24
    - 1.5190853364638501
    top_right:
    - -58.193333333333335
    - 1.5190853364638501
    bottom_right:
    - -58.193333333333335
    - -12.79594015098676
    bottom_left:
    - -45.24
    - -12.79594015098676
  center:
  - -51.71666666666667
  - -5.638427407261455
  width: 12.953333333333333
  height: 14.315025487450612
- id: region_11
  speed_gain: 3.0
  steering_gain: 1.0
  corners:
    top_left:
    - 60.41666666666666
    - -16.471419668034887
    top_right:
    - 50.653333333333336
    - -16.471419668034887
    bottom_right:
    - 50.653333333333336
    - -39.007912496251045
    bottom_left:
    - 60.41666666666666
    - -39.007912496251045
  center:
  - 55.535
  - -27.739666082142968
  width: 9.763333333333321
  height: 22.536492828216158
